app/main.py

```diff
@@ -1,6 +1,21 @@
 from typing import Callable
+from functools import wraps
+from typing import Any, Dict, Tuple
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store: Dict[Tuple, Any] = {}
+
+    @wraps(func)
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
+        key = (args, frozenset(kwargs.items()))
+        if key in cache_store:
+            print("Getting from cache")
+            return cache_store[key]
```

Comment:
delete this line

Reply:
i don`t understand, but then it doesn`t work correctly..., could you clarify more clear?

Reply:
you duplicate return here and in 19 line, you can avoid this